devtools/environment.yml

```diff
@@ -3,9 +3,8 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - MDAnalysis<2.0.0
-  - MDAnalysisTests<2.0.0
-  - six
+  - MDAnalysis>2.0.0
+  - MDAnalysisTests>2.0.0
```

Comment:
Shouldn't this be `>=` because any 2.x should work

Reply:
It should